connectors-home/modules/ROOT/pages/introduction/intro-config-oauth2.adoc

```diff
@@ -0,0 +1,189 @@
+= Configuring OAuth 2.0 for Connectors
+
+Many services enable you to use an OAuth 2.0 authentication scheme for increased security when interacting with their APIs. Many connectors support configuring a standard OAuth 2.0 setup.
+
+== Prerequisites
+
+To configure OAuth 2.0 for a connector, you must have access to the following:
+
+* OAuth provider (Okta, Azure AD, Google, PingFederate, and so on)
+* Anypoint Studio
+* Any connector that supports OAuth 2.0
+
+== How to Configure OAuth 2.0 For a Connector
```

Comment:
This is not really a "how to configure" section but just "how it works" section. I wanted to provide the user with high level understanding of what's actually happening when they use OAuth2 because the concept is not obvious to non-technical people. 🤔 If we want to avoid a title like "how it works" then maybe "OAuth 2.0 Flow overview" would be more appropriate.

Also if you think this isn't the appropriate place for it. I think we can also move it bellow the "OAuth 2.0 Configuration Fields" or even completely at the end. We could also drop it completely, it's not vital, but I wanted to explain to people what they should expect, especially since this becomes even more convoluted once CloudHub is involved in to second doc.

Essentially this section just describes [this image of the OAuth 2.0 flow](https://developers.google.com/static/identity/protocols/oauth2/images/flows/authorization-code.png).

What do you think @sathya0 ?

Reply:
Great point @Hazarth! I agree with you. I changed the docs to match your feedback.